app/templates/mandatarissen/persoon/mandataris.hbs

```diff
@@ -27,41 +27,40 @@
           )}}
           @tooltipText={{this.toolTipText}}
         >
-          <AuButton
-            @disabled={{(or
-              (not this.model.mandataris.isActive)
-              this.isDisabledBecauseLegislatuur
-            )}}
-            {{on "click" (fn (mut this.isChanging) true)}}
-          >Wijzig huidig mandaat</AuButton>
+          <Shared::Tooltip
+            @showTooltip={{(not this.isDisabledBecauseLegislatuur)}}
```

Comment:
je checkt hier maar een van de twee condities, als je een niet actieve mandataris wenst aan te passen krijg je een dubbele tooltip 
![Screenshot 2024-10-22 at 14 08 23](https://github.com/user-attachments/assets/836919c6-e5d4-4124-b24d-3b958859645e)

Reply:
Fixed non chaning of tooltips + double tooltip